mood-bot/src/main/java/io/beekeeper/mood_bot/ProfileIterator.java

```diff
@@ -0,0 +1,63 @@
+package io.beekeeper.mood_bot;
+
+import java.util.Iterator;
+import java.util.List;
+
+import io.beekeeper.sdk.BeekeeperSDK;
+import io.beekeeper.sdk.exception.BeekeeperException;
+import io.beekeeper.sdk.model.UserProfile;
+
+public class ProfileIterator {
```

Comment:
This class seems low-level enough to be provided as a utility on the SDK level, right?

Reply:
With a bit of polish, yeah, let's do that